### DISCUSSION TITLE
Issues faced while cloning a project

### DISCUSSION THREAD

**Discussion by Krishnanunni2001** [community/community#46705] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://github.com/Danny1024-na/amazonClone
I forked this project and clone it in my git hub desktop.
But it didn't worked properly.
The files which are showed in the repo is not found when it cloned.
It is very difficult to find the files and understand the code ,similarly I cloned some other projects but  the same thing happened.
Can you please help me out.

**Comment by Synceratus** [comment]
Are you sure you're cloning it the right way? Take a look at this detailed guide:
https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop
